src/response.rs

```diff
@@ -96,38 +97,103 @@ impl fmt::Debug for Response {
             "{}",
             match *self {
                 // IData
-                GetIData(..) => "Response::GetIData",
+                GetIData(ref res) => format!("Response::GetIData => {:?}", ErrorDebug(res)),
```

Comment:
I have a slight preference for @madadam's original formatting suggestion, i.e. not having the ` => `, but rather wrapping the result in brackets:
```rust
format!("Response::GetIData({:?})", ErrorDebug(res))
```
since it's a bit more space efficient, but also better represents the response type.  It's not a big deal though - if you have a strong preference for the current style, feel free to ignore this :)